packages/http-helper/src/sec_fetch/sec_fetch_mode.ts

```diff
@@ -0,0 +1,10 @@
+//  @see
+//  - https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Sec-Fetch-Mode
+//  - https://w3c.github.io/webappsec-fetch-metadata/#sec-fetch-mode-header
+//  - https://fetch.spec.whatwg.org/#concept-request-mode
+
+export const CORS = 'cors' as const;
+export const NAVIGATE = 'navigate' as const;
+export const NO_CORS = 'no-cors' as const;
+export const SAME_ORIGIN = 'same-origin' as const;
+export const websocket = 'websocket' as const;
```

Comment:
変数名の方の websocket が小文字なのは何か意図がありますか？

Reply:
This sorts to valued defined by spec:
https://w3c.github.io/webappsec-fetch-metadata/#sec-fetch-mode-header

Reply:
Above comment intends to mention about variable name not value. I'd think the below suggestion alignts to the other variable name.

```suggestion
export const WEBSOCKET = 'websocket' as const;
```

Reply:
addressed in https://github.com/Nikkei/node-http-helper/pull/190/commits/ce75213394128db822d5e5d15ac040559359aac9